rook/run_query/entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -eu
+
+# export the lib path
+export LD_LIBRARY_PATH=/usr/lib64/ceph
+
+# install required packages
+yum -y update
+yum install -y wget bc
```

Comment:
we can use `curl` and do math using `$(( ))`, so that we don't have to install stuff when as part of the execution of the step

Reply:
Turns out `$(())` only handles integer calculations. So to preserve precision maybe we need to use `bc`